src/__namespace.js

```diff
@@ -1,4 +1,5 @@
 /**
  * @ignore
  */
-wikibase.datamodel = wikibase.datamodel || {};
+window.wikibase = window.wikibase || {};
```

Comment:
As far as I understood it might be PhntomJS "feature", but any way new code should work just fine.